app/src/app/create/create.service.ts

```diff
@@ -30,9 +30,9 @@ import { NavStepStatus } from './navtree/navstep.status';
 export const CREATE_TREE = new InjectionToken<NavTreeNode>('create.tree');
 
 export const APP_CREATE_TREE: NavTreeNode =
-  new LinearNavTreeNode('general', 'Generalities',
-    new LinearNavTreeNode('simpleAlternatives', 'Alternatives',
-      new FinalNavTreeNode('round', 'Rounds')
+  new LinearNavTreeNode('general', '. Generalities',
+    new LinearNavTreeNode('simpleAlternatives', '. Alternatives',
+      new FinalNavTreeNode('round', '. Rounds')
```

Comment:
No. This is not a file about how things should be displayed. This part of the code describes the steps of the creation. Having a step called '. Generalities' does not make sense at all. Moreover, this presentation feature is repeated, which is bad. The point and the space must be added when this information is displayed, in the HMTL or in the SASS file. For instance, you could replace `{{ step }}` with `. {{ step }}` in the HTML file. Or you could keep the `index:after { content: ". " }` that was in the style file.